.github/workflows/static.yml

```diff
@@ -9,54 +9,59 @@ on:
 jobs:
   phpstan:
     name: PHPStan
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
 
     steps:
-      - name: Set up PHP
-        uses: shivammathur/setup-php@2.7.0
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
         with:
-          php-version: '7.4'
+          php-version: 7.4
           coverage: none
+          tools: phpstan:1.4.6, cs2pr
 
       - name: Checkout code
         uses: actions/checkout@v2
 
       - name: Download dependencies
-        run: composer install --no-interaction --optimize-autoloader
+        uses: ramsey/composer-install@v1
 
       - name: PHPStan
-        uses: docker://oskarstark/phpstan-ga:0.12.48
-        with:
-          entrypoint: /composer/vendor/bin/phpstan
-          args: analyze --no-progress
+        run: phpstan analyze --no-progress --error-format=checkstyle | cs2pr
```

Comment:
no need to use `--error-format=checkstyle | cs2pr`. phpstan supports github annotations natively since a long time.

Reply:
Yeah, I know. But there is a slight difference last I checked. I dont remember what it was now.